src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/RuntimeInformation.Browser.cs

```diff
@@ -5,7 +5,13 @@ namespace System.Runtime.InteropServices
 {
     public static partial class RuntimeInformation
     {
+#if TARGET_WASI
         public static string OSDescription => "Browser";
+#elif TARGET_BROWSER
+        public static string OSDescription => "WASI";
```

Comment:
cases are swapped:

```suggestion
#if TARGET_BROWSER
        public static string OSDescription => "Browser";
#elif TARGET_WASI
        public static string OSDescription => "WASI";
```

Reply:
we should also probably rename the file to RuntimeInformation.wasm.cs

Reply:
Thanks for the fix. Re: file rename, this PR is not trying to fix managed part. There are many other .cs files which will need to be renamed and improved.